### PR TITLE
gdiObjects.xBar version 2.0.2112.20

### PR DESCRIPTION
**Pull request by gdiObjects**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/denelon/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/38368)

### CODE REVIEW

**Review by ItzLevvie — APPROVED** [review]

File: manifests/g/gdiObjects/xBar/2.0.2112.20/gdiObjects.xBar.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+PackageUrl: https://www.gdiobjects.com/Products/xBar/Index.php
+License: Copyright © 2008-2021 gdiObjects
+ShortDescription: A Dockable Infobar and App Launcher for Windows
+Description: The Fast & Easy Way to Find and Launch Software Apps, Open Documents, or Access Folders, and Drives in Windows. Quickly Organize Shortcuts into User-defined Categories. View System Details and Monitor your PC’s Performance with CPU, Memory, Temperature, and Battery Sensors.
+Moniker: xBar
+Tags:
+- sidebar
+- appbar
+- windows 10
+- windows 11
```
Comment (ItzLevvie) [inline manifests/g/gdiObjects/xBar/2.0.2112.20/gdiObjects.xBar.locale.en-US.yaml:22]:
```suggestion
- windows-10
- windows-11
```

File: manifests/g/gdiObjects/xBar/2.0.2112.20/gdiObjects.xBar.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: gdiObjects.xBar
+PackageVersion: 2.0.2112.20
+Installers:
+- Architecture: x86
```
Comment (ItzLevvie) [inline manifests/g/gdiObjects/xBar/2.0.2112.20/gdiObjects.xBar.installer.yaml:7]:
```suggestion
- Architecture: x64
```

File: manifests/g/gdiObjects/xBar/2.0.2112.20/gdiObjects.xBar.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+PublisherUrl: https://www.gdiobjects.com/index.php
+PublisherSupportUrl: https://www.gdiobjects.com/helpdesk/
+PrivacyUrl: https://www.gdiobjects.com/privacy.php
+Author: gdiObjects
+PackageName: xBar
+PackageUrl: https://www.gdiobjects.com/Products/xBar/Index.php
+License: Copyright © 2008-2021 gdiObjects
+ShortDescription: A Dockable Infobar and App Launcher for Windows
+Description: The Fast & Easy Way to Find and Launch Software Apps, Open Documents, or Access Folders, and Drives in Windows. Quickly Organize Shortcuts into User-defined Categories. View System Details and Monitor your PC’s Performance with CPU, Memory, Temperature, and Battery Sensors.
+Moniker: xBar
```
Comment (ItzLevvie) [inline manifests/g/gdiObjects/xBar/2.0.2112.20/gdiObjects.xBar.locale.en-US.yaml:17]:
```suggestion
Moniker: xbar
```
This is meant to be in lower-case.

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

File: manifests/g/gdiObjects/xBar/2.0.2112.20/gdiObjects.xBar.locale.en-US.yaml (hunk 1 above)
Comment (Trenly) [reply manifests/g/gdiObjects/xBar/2.0.2112.20/gdiObjects.xBar.locale.en-US.yaml]: I would argue that these tags should be removed entirely, as they are what I would consider to be "low-value" tags, in that they don't provide any meaningful information about the application itself or it's uses. They simply denote which versions of windows the application can be installed on, which could easily be covered by adding the MinimumOSVersion

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**